### ISSUE TITLE
Open certain links in selected Tab Group in Safari

### ISSUE THREAD

**Issue by travellingdeon** [johnste/finicky#317]
I use a lot of different tab groups for different things I am working on/playing with.

I'd love to be able to get Finicky to direct certain links to open in certain Tab Groups.

**Comment by reefdog** [comment]
I believe this is a duplicate of #296, yes?

And @travellingdeon out of curiosity, now that Safari 17 has [profiles](https://support.apple.com/en-us/105100), do you still want the ability to target tab groups? (Profiles seems a more natural fit for this sort of need, at least in my own experience.)

**Comment by gegoune** [comment]
I, for one, only use different browser for work related browsing due to finicky's lack of profile support for Safari. 

But to answer your question to OP: yes, having both, profile and tab group, would be very beneficial, allowing more targeted workflows. 

Does Safari expose what's required for this functionality to land in fincky?

**Comment by reefdog** [comment]
> I, for one, only use different browser for work related browsing due to finicky's lack of profile support for Safari.

Historically, same. (Safari/personal, Firefox/work.) I've been experimenting with Safari profiles instead, but the inability to route to specific Safari profiles will be a real drag.

> Does Safari expose what's required for this functionality to land in fincky?

Unknown.

**Comment by travellingdeon** [comment]
Yes, definitely still want to open in tab groups. I have 6 different tab groups that I use for work, with a lot of new tasks coming from Slack. When a certain type of link comes, I want it directed to the right browser, or the right tab group in ***@***.***> wrote:﻿
I believe this is a duplicate of #296, yes?
And @travellingdeon out of curiosity, now that Safari 17 has profiles, do you still want the ability to target tab groups? (Profiles seems a more natural fit for this sort of need, at least in my own experience.)

**Comment by mariusaee** [comment]
I'm really waiting for this feature too!

**Comment by idengrenme** [comment]
> I use a lot of different tab groups for different things I am working on/playing with.
> 
> I'd love to be able to get Finicky to direct certain links to open in certain Tab Groups.

I use Chrome for work (Safari for "play") and have TONS of tab groups in Chrome - can Finicky support tab groups in Chrome?

**Comment by johnste** [comment]
Unfortunately as far as I can tell Safari doesn't provide any methods of opening links in specific tab groups.

**Comment by khomaldi** [comment]
> Unfortunately as far as I can tell Safari doesn't provide any methods of opening links in specific tab groups.

how about chrome?
